Cargo.toml

```diff
@@ -35,6 +35,7 @@ cached = "0.24.1" # 0.25.x versions force `syn` dependency version to be 1.0.66
 float-cmp = "0.9.0"
 lazy_static = "1.4.0"
 rand = "0.8.3"
+lexical = "6.1.1"
```

Comment:
Why is a new package needed? Is it not enough with `f64::from_str`?

Reply:
Sorry for the late reply. I thought by mistake that `f64::from_str` was not enough. I fixed it.